python/ray/util/sgd/v2/constants.py

```diff
@@ -25,3 +27,6 @@
 
 # Default filename for JSON logger
 RESULT_FILE_JSON = "results.json"
+
+# Default directory where all SGD logs, checkpoints, etc. will be stored.
+DEFAULT_RESULTS_DIR = Path("~/ray_results").expanduser()
```

Comment:
This is also the log directory for rllib right? Do you know if there is confusion between rllib and tune today? Another idea would be to make this `~/ray_results/sgd` but we don't have a similar hierarchy for tune so this might also be confusing.

In general I don't think this should be a problem because by default the trainer logdir will be prefixed with `sgd_` and can easily be identified with:
```bash
ls ~/ray_results | grep sgd_
```

Reply:
Gotcha. A prefix is completely sufficient. Thanks!

Reply:
BTW `~/` doesn't work on windows, just a heads up